TypeCobol.LanguageServer.Test/LSRTests/ExecuteCommandRefactorAdjustFillers_WithCopy/input/CopyFolder/YMYCOPY.cpy

```diff
@@ -0,0 +1,6 @@
+       01 :DVZOSM:.
+           05 :DVZOSM:-Zone pic X(120).
+           05 :DVZOSM:-Zone-Redef REDEFINES :DVZOSM:-Zone.
+              10 Var1 PIC X(20).
+      * Should be 100 but won't be altered by refactor on including pgm
```

Comment:
As this adjusted FILLER should have the same length as the program one, it is not obvious that the result change is related to the program.
Is it possible to modify the redefined zone's length?

Reply:
Done